### PR TITLE
SECACC-1514: Add button loading state

### PR DESCRIPTION
**Pull request by the-sky-alight**
**What**:

**Why**:

**How**:

**Checklist**:

* [ ] Documentation
* [ ] Tests
* [ ] Ready to be merged

### CODE REVIEW

File: src/components/Button/Loader/styles.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  display: inline-block;
+  width: 6px;
+  height: 6px;
+  border-radius: 50%;
+  background: ${({ variant }) => colorVariants[variant]};
+  animation: ${wave} 0.8s linear infinite;
+
+  &:nth-child(2) {
+    animation-delay: -0.1s;
+    margin: 0 8px;
```
Comment (nyonchev) [inline src/components/Button/Loader/styles.js]: Do we have arora constant for 8px? If yes we should use it here

File: src/components/Button/Loader/index.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { Dot, LoaderWrapper, ContentWrapper, DotsWrapper } from "./styles";
+import { BUTTON_VARIANTS } from "../../constants";
+
+const Loader = ({ children, loading, variant }) => (
```
Comment (stremann) [inline src/components/Button/Loader/index.js:7]: Should `Loader` be placed under `Button` component? What do you think to have it under independent LoaderDot/ like LoaderRing/ already presented in Aurora?

Comment (the-sky-alight) [reply src/components/Button/Loader/index.js:7]: From my point of view the Loader cannot exist by itself and should be used as a part of bigger component. Also, the Button is the only component which requires these dots. Could be discussed further if needed.

Comment (the-sky-alight) [reply src/components/Button/Loader/index.js:7]: Discussed.

File: src/components/Button/Loader/styles.js (hunk 1 above)
Comment (the-sky-alight) [reply src/components/Button/Loader/styles.js]: Updated.

**Pull request merged**